cwatm/management_modules/data_handling.py

```diff
@@ -1175,6 +1175,7 @@ def readnetcdf2(namebinding, date, useDaily='daily', value='None', addZeros = Fa
         if cutcheckmask == cutcheckmap: cut = False
 
     if cut:
+        cutmap[0], cutmap[1], cutmap[2], cutmap[3] = mapattrNetCDF(name)
```

Comment:
This looks great! For the cut commit, we'd like to include this as an option that can be activated. For example, by including the two lines beforehand:

```
if 'maps_cut_individually' in option:
    if checkOption('maps_cut_individually'):
        cutmap[0], cutmap[1], cutmap[2], cutmap[3] = mapattrNetCDF(name)
```

Would you include these two lines?

Then, in the OPTIONS section of the settings file, one could `include maps_cut_individually = True`, and the new code chunk would be run.

Reply:
Of course! Thanks, Mikhail.